File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";

        String[] alpha={"a","b","c","d","e","f","g","h","i","j"};
        String s=String.valueOf(age);
        for(char x: s.toCharArray()){
            int n=x-'0';
            answer+=alpha[n];
        }
        return answer;
    }
}

File: 프로그래머스/unrated/120834. 외계행성의 나이/외계행성의 나이.swift
import Foundation

func solution(_ age:Int) -> String {
    
    var alpha = ["a","b","c","d","e","f","g","h","i","j"]
    
    var answer = ""
    
    var arr = Array(String(age).compactMap{ Int(String($0))})
    for i in arr {
        answer.append(alpha[i])
    }
    
    return answer
}
